src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/NameSyntaxClassifier.cs

```diff
@@ -257,6 +257,13 @@ private static string GetClassificationForLocal(ILocalSymbol localSymbol)
 
         private static string GetClassificationForMethod(IMethodSymbol methodSymbol)
         {
+            if (methodSymbol.MethodKind == MethodKind.Constructor)
+            {
+                return methodSymbol.ContainingType.TypeKind == TypeKind.Class
+                    ? ClassificationTypeNames.ClassName
+                    : ClassificationTypeNames.StructName;
+            }
```

Comment:
1. why no check for constructor?
2. you assume ContainingType is never null here, but you check for null above.  Why teh inconsistency?
3. Can we share any of this logic?

Reply:
I added a comment about destructors. I was just matching the surrounding code, I've updated to it is more consistent.